modules/person.rb

```diff
@@ -0,0 +1,35 @@
+require_relative './nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown')
+    super()
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = true
+    @rentals = []
+  end
+
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  def correct_name
+    name
+  end
+
+  def rent_book(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Same for this method kindly consider using the ` Rentals` class to create a new rental 

```suggestion
  def rent_book(rental)
    Rental.new(date, book, self)
  end
```